src/pymovements/gaze/io.py

```diff
@@ -311,6 +311,86 @@ def from_csv(
     return gaze
 
 
+def _metadata_to_cal_val_frames(metadata: dict[str, Any]) -> tuple[pl.DataFrame, pl.DataFrame]:
```

Comment:
it's probably best to separate these into `metadata_to_cal_frame` and `metadata_to_val_frame`